crypto/evp/digest.c

```diff
@@ -587,6 +587,8 @@ static void *evp_md_from_dispatch(const OSSL_DISPATCH *fns,
     if ((md = EVP_MD_meth_new(NID_undef, NID_undef)) == NULL)
         return NULL;
 
+    md->name = OPENSSL_strdup(name);
```

Comment:
Should this be NULL checked?
Not having the name isn't critical I guess.

Reply:
Fixing

Reply:
Missed this one?

Reply:
Indeed I did.  Also, I will add an error report on all of them.